app/views/admin/users/_bid_history.html.erb

```diff
@@ -4,8 +4,8 @@
   <thead>
     <tr>
       <th scope="col">Auction</th>
-      <th scope="col">Start Date</th>
-      <th scope="col">End Date</th>
+      <th scope="col">Status</th>
+      <th scope="col">Skills</th>
```

Comment:
Okay. It looks like we sort the auctions in descending chronological order so I guess that'll be okay